geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/TestWithKerberos.scala

```diff
@@ -0,0 +1,61 @@
+/** *********************************************************************
+  * Crown Copyright (c) 2016 Dstl
+  * All rights reserved. This program and the accompanying materials
+  * are made available under the terms of the Apache License, Version 2.0
+  * which accompanies this distribution and is available at
+  * http://www.opensource.org/licenses/apache2.0.php.
+  * ************************************************************************/
+
+package org.locationtech.geomesa.accumulo
+
+import org.specs2.mutable.Specification
+
+import org.apache.kerby.kerberos.kerb.server.SimpleKdcServer
+import org.apache.kerby.util.NetworkUtil
+
+import java.io.File
```

Comment:
As a picky comment, we use IntelliJ's 'optimize imports' to sort imports in alphabetically order (and remove the spaces between the lines).